experimental/Schemes/Types.jl

```diff
@@ -475,8 +475,8 @@ Glueing of two affine schemes ``X ↩ U ≅ V ↪ Y`` along open subsets
     ambient(domain(f)) === X || error("the domain of the glueing morphism is not an open subset of the first argument")
     ambient(codomain(f)) === Y || error("the codomain of the glueing morphism is not an open subset of the second argument")
     if check
-      (domain(f) === codomain(g) &&
```

Comment:
Why revert these changes? I think `===` was just the right thing here! This is what caused the other bug that @simonbrandhorst asked be to look at above.

Reply:
Alright. So this has been a mistake previously. I will do it, once the tests have passed, so that we at least once have a result on test coverage.

Reply:
The coverage seems to be okay. It slightly increased. I would defer any further coverage increases to future PRs.

Reply:
O.k. I reverted it. Let's see what the tests say.